python/ray/dashboard/tests/test_dashboard.py

```diff
@@ -1209,62 +1209,6 @@ async def test_dashboard_module_load(tmpdir):
     } == loaded_subprocess_module_handles_expected
 
 
-@pytest.mark.skipif(
-    os.environ.get("RAY_MINIMAL") == "1",
-    reason="This test is not supposed to work for minimal installation.",
-)
-def test_extra_prom_headers_validation(tmpdir, monkeypatch):
```

Comment:
This has been replaced by the unit test `test_parse_prom_headers`. We can't catch the `ValueError` in the parent process, and this test effectively just tested the functionality of `test_parse_prom_headers`.